tests/expr_and_series/clip_test.py

```diff
@@ -28,8 +28,6 @@ def test_clip_expr(
 def test_clip_expr_expressified(
     request: pytest.FixtureRequest, constructor: Constructor
 ) -> None:
-    if "modin_pyarrow" in str(constructor):
-        request.applymarker(pytest.mark.xfail)
     if "cudf" in str(constructor):
         # https://github.com/rapidsai/cudf/issues/17682
         request.applymarker(pytest.mark.xfail)
```

Comment:
I would assume that the same trick might work for cudf. I will try it tomorrow on kaggle and update it here of that's the case